src/test/java/com/gluonhq/substrate/SubstrateTest.java

```diff
@@ -89,4 +91,16 @@ void testMainClassName() {
         config = new InternalProjectConfiguration( new ProjectConfiguration("name/a.b.Foo"));
         assertEquals("a.b.Foo", config.getMainClassName());
     }
+
+    @Test
+    void testAssertSW() {
+        ProjectConfiguration publicConfig = new ProjectConfiguration("a.b.Foo");
+        InternalProjectConfiguration config = new InternalProjectConfiguration(publicConfig);
+        assertFalse(config.isUsePrismSW());
+
+        publicConfig = new ProjectConfiguration("a.b.Foo");
+        publicConfig.setUsePrismSW(true);
+        config = new InternalProjectConfiguration(publicConfig);
+        assertTrue(config.isUsePrismSW());
+    }
```

Comment:
test on false too?

Reply:
It is tested [here](https://github.com/gluonhq/substrate/pull/290/files#diff-4b102605198c43c8c619729d6fa876bdR99)